train.py

```diff
@@ -261,8 +291,22 @@ def train():
     print('Begin training!')
     print()
     # try-except so you can use ctrl+c to save early and stop training
+    # SparseML Integration
+    half_precision = args.fp16 and args.device != 'cpu'
+    scaler = amp.GradScaler(enabled=half_precision)
+    sparseml_wrapper.initialize_loggers(logger, tb_writer=TensorBoardLogger,
+                                        wandb_logger=None, rank=-1)
+    scaler = sparseml_wrapper.modify(scaler, optimizer, net, data_loader)
```

Comment:
as discussed before, let's wrap `yolact_net` instead of `net` here so we don't have to deal with unwrapping the `module.net` in recipes

Reply:
Done!

Reply:
Will sync Offline for this issue!